extractor/src/extractors/JsonnetfileJsonExtractor.ts

```diff
@@ -0,0 +1,40 @@
+import MatchConfig from "../matcher/MatchConfig";
+import Extractor from "./Extractor";
+import ExtractorFile from "./ExtractorFile";
+import Languages from "./Languages";
+
+const extract = (dependency: any,scopes:any) => {
```

Comment:
Remove scopes as a parameter. Just use the subdir if it's set. Otherwise, it should be an empty array.

Reply:
Got it , I have a small question about : 
`dependency.source.git.remote` and `dependency.source.git.subdir` should i check all the navigation like : `if ( dependency.source && dependency.source.git && dependency.source.git.remote )` or are we sure`dependency.source.git` is truthy ?